lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/filteredStringListView.py

```diff
@@ -57,47 +73,58 @@ def __init__(self, items: Sequence[str] = None,  headerTitle:str = "", parent=No
         self.setTextElideMode(Qt.TextElideMode.ElideMiddle)
         self.setSelectionBehavior(QListView.SelectRows)
         self.setSelectionMode(QListView.MultiSelection)
-        self.setContentsMargins(0,0,0,0)
+        self.setContentsMargins(0, 0, 0, 0)
 
-        self.selectionModel().selectionChanged.connect(lambda: self.selectedItemsChanged.emit())
+        self.setCursor(Qt.ArrowCursor)
 
-        self.placeholder_label = QLabel("Drag objects here or click “+” to add", self)
-        self.placeholder_label.setAlignment(Qt.AlignmentFlag.AlignCenter)
-        self.placeholder_label.setStyleSheet("color: gray; font-size: 18px;")
-        self.placeholder_label.hide()
+        self.selectionModel().selectionChanged.connect(
+            lambda: self.selectionChanged.emit()
+        )
 
     def drawFrame(self, painter: QPainter):
         pass
 
-    def paintEvent(self, event: QPaintEvent):
-        super(FilteredStringListView, self).paintEvent(event)
-        if self._model.isFilteredEmpty():
-            painter = QPainter(self.viewport())
-            painter.setPen(QColor(128, 128, 128))
-            painter.drawText(self.rect(), Qt.AlignCenter, kNoObjectFoundLabel)
-
```

Comment:
Both the paint-override method and the place-holder label method (which were redundant with each other...) are now removed. What is replacing these the place-holders?